_includes/contact_form.html

```diff
@@ -381,12 +385,8 @@
     <span class="text-bold"
```

Comment:
I think this extra span was only necessary when we had the "Optional" label. We could probably unwrap it and remove the `text-normal` class on the label.